tutor/templates/apps/openedx/config/lms.env.json

```diff
@@ -27,6 +27,7 @@
   "CMS_BASE": "{{ CMS_HOST }}",
   "LMS_BASE": "{{ LMS_HOST }}",
   "CONTACT_EMAIL": "{{ CONTACT_EMAIL }}",
+  "CONTACT_MAILING_ADDRESS": "{{ PLATFORM_NAME }}@{{ LMS_HOST }}",
```

Comment:
Can I suggest moving this settings to the py settings? I'd rather avoid adding stuff the the json config files, when possible. This setting will be necessary both in the LMS and the CMS, so you can add it to the common partial settings file.

Reply:
Yes, no problem. What's a good rule of thumb to make decisions when it comes to where configurations settings should be placed?

Reply:
Some values from the json files are parsed in the edx-platform upstream settings files to define multiple setting values. For instance, `PLATFORM_NAME` is used to define the `CC_MERCHANT_NAME` setting. In such cases, it's better to add the values to the env.json files to avoid code duplication.

Some values must be present in the env.json files, otherwise causing a crash. This is the case of the `CACHES` variable for instance.

Feature flags are also defined in the env.json files because... well I'm not sure. Some feature flags are also used in multiple places in the settings. So if we defined all feature flags in the settings modules, it would cause some duplication. I'd rather avoid splitting feature flags across both the json and the py settings files.

In all other cases, it's better if settings are defined directly in the python modules, to avoid code indirections.